diff -up bridge-utils-1.5/libbridge/libbridge.h.in6_addr bridge-utils-1.5/libbridge/libbridge.h
--- bridge-utils-1.5/libbridge/libbridge.h.in6_addr	2011-03-29 02:52:54.000000000 +0200
+++ bridge-utils-1.5/libbridge/libbridge.h	2013-01-13 15:36:59.708082794 +0100
@@ -20,6 +20,7 @@
 #define _LIBBRIDGE_H
 
 #include <sys/socket.h>
+#include <linux/in6.h>
 #include <linux/if.h>
 #include <linux/if_bridge.h>
 
